client/overview/modal/connection-success/index.tsx

```diff
@@ -18,15 +18,12 @@ export const ConnectionSuccessModal = () => {
 	const [ isDismissed, setIsDismissed ] = React.useState(
 		wcpaySettings.isConnectionSuccessModalDismissed
 	);
-	const { updateOptions } = useDispatch( 'wc/admin/options' );
 
 	const onDismiss = async () => {
 		setIsDismissed( true );
 
 		// Update the option to mark the modal as dismissed.
-		await updateOptions( {
-			wcpay_connection_success_modal_dismissed: true,
-		} );
+		await saveOption( 'wcpay_connection_success_modal_dismissed', true );
```

Comment:
Nitpick: We can remove `await` here.

Reply:
Addressed ecbdedc02a0db39079b154696d1fc2669d10f541